crates/swift-bridge-ir/src/codegen/codegen_tests/result_codegen_tests.rs

```diff
@@ -407,11 +407,15 @@ mod extern_rust_fn_return_result_opaque_rust_type_and_transparent_enum_type {
         }
     }
 
+    // In Rust 1.79.0 dead_code warnings are issued for wrapped data in enums in spite of the enum
+    // having `#[repr(C)]`.
```

Comment:
Let's:

- link to the Rust issue
- make it clear that this can be deleted when that issue is closed

This way no matter where you run into this `#[allow(unused)` you can easily check whether it can be removed.